### ISSUE TITLE
Body scanned script broken

### ISSUE THREAD

**Issue by EeeDeeBee** [EDCD/EDDI#1988]
## EDDI version in which issue is found
v.3.7.0

## VoiceAttack version in which issue is found (as applicable)
[Skip this section if VoiceAttack was not running when the issue occurred. Please give the full version number here, not just "latest" or "current"]

## Steps to reproduce
1. Install and open EDDI
2. Either test the "Body scanned" script, or scan a body in a system to trigger it in game

## Expected
Some kind of scan info to be read out

## Observed
An exception is read out instead. The full error from the log file is:

```
2020-11-05T15:33:48 [Warning] ScriptResolver:resolveFromValue Exception has been thrown by the target of an invocation. System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at EddiDataDefinitions.Ship.get_size()
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
   at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Cottle.Values.ReflectionValue.MemberReader.<>c__DisplayClass6_0.<.ctor>b__0(Object s)
   at Cottle.Values.ReflectionValue.MemberReader.Extract(Object source)
   at Cottle.Values.ReflectionValue.Resolve()
   at Cottle.Values.ResolveValue.Acquire()
   at Cottle.Values.ResolveValue.get_Fields()
   at Cottle.Documents.Simple.Evaluators.AccessEvaluator.Evaluate(IStore store, TextWriter output)
   at Cottle.Documents.Simple.Evaluators.AccessEvaluator.Evaluate(IStore store, TextWriter output)
   at Cottle.Documents.Simple.Evaluators.InvokeEvaluator.Evaluate(IStore store, TextWriter output)
   at Cottle.Documents.Simple.Evaluators.InvokeEvaluator.Evaluate(IStore store, TextWriter output)
   at Cottle.Documents.Simple.Nodes.IfNode.Render(IStore store, TextWriter output, Value& result)
   at Cottle.Documents.Simple.Nodes.CompositeNode.Render(IStore store, TextWriter output, Value& result)
   at Cottle.Documents.SimpleDocument.Render(IStore store, TextWriter writer)
   at Cottle.Documents.AbstractDocument.Render(IStore store)
   at EddiSpeechResponder.ScriptResolver.resolveFromValue(String script, BuiltinStore store, Boolean isTopLevelScript, Script scriptObject)
```

## Investigation
Restarting Elite Dangerous or EDDI in any order did not help. Reinstalling EDDI resulted in the same error again.

**Comment by EeeDeeBee** [comment]
About 30 minutes later I opened EDDI to try again and it now works as expected, although it skips reading out scan info seemingly at random.

I changed no settings.

**Comment by Tkael** [comment]
The exception appears to be a duplicate of #1984, a known intermittent issue that can occur after EDDI starts and that tends to self correct from in-game activity. A fix has already been implemented and will be included with the next release (soon™).

I would need more information to respond to the second concern about scan info being sometimes skipped?

**Comment by Tkael** [comment]
Another user reported today that they had more success with exiting to Elite's main menu and returning.

**Comment by Tkael** [comment]
The exception is believed to be fixed in version 3.7.1-b1, released today and available from https://github.com/EDCD/EDDI/releases.

If you can provide more detail **on a separate ticket** about the second concern with scan info sometimes being skipped, we'll be happy to look into that concern further.
